### PR TITLE
:bug: lrucache.get should not ignore false values

### PR DESCRIPTION
**Pull request by feisuzhu**
Addresses https://github.com/openresty/lua-resty-lrucache/issues/19

### CODE REVIEW

**Comment by agentzh** [comment]
@feisuzhu Seems like the Travis CI tests on your branch here all failed:

https://travis-ci.org/openresty/lua-resty-lrucache/builds/214112082

Also, will you please add some corresponding tests to the existing test suite to cover your fix? Thanks!

**Comment by feisuzhu** [comment]
Finally got it right, embarrassing...

**Comment by agentzh** [comment]
@feisuzhu Will you make sure the `resty.lrucache.ffi` module in this library also works the same way here? Thanks!

**Comment by feisuzhu** [comment]
pureffi version is not affected :)

File: t/init-by-lua.t [hunk 1]
```diff
@@ -55,6 +55,12 @@ __DATA__
             c:delete("cat")
             log("dog: ", c:get("dog"))
             log("cat: ", c:get("cat"))
+
+            c:set("false-value", false)
+            log("false-value: ", c:get("false-value"))
+
+            c:delete("false-value")
+            log("false-value: ", c:get("false-value"))
```
Comment (agentzh) [inline t/init-by-lua.t]: I think these tests deserve a separate test case? Or even a separate test file like `bug.t`. The existing test case you are patching has a specific indentation of testing the library in the context of `init_by_lua*`, it's not a good idea to hijack test cases with a different intention just for the sake of convenience ;) Will you rework the tests accordingly? Thanks!

**Comment by feisuzhu** [comment]
Reworked :)

**Comment by agentzh** [comment]
@feisuzhu Great. Thank you! Looking good now.

@yangshuxin @doujiang24 Will you have a look at this too? Thanks!

**Comment by agentzh** [comment]
@feisuzhu One small thing: you should use the `bug: ` prefix instead of `:bug: ` in your commit log messages. Being consistent is very important no matter how much you like your own style ;) I'll fix that on my side this time. Thanks!

**Comment by agentzh** [comment]
BTW, I changed your commit log message to the following:

```
bug: resty.lrucache: the get() method incorrectly ignored `false` values. #19
```

**Comment by agentzh** [comment]
Merged with minor edits. Thanks!

**Pull request closed without merging**